addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -44,15 +44,16 @@ def _onchange_partner_ids(self):
                         ', '.join(invalid_partners.mapped('name'))
                     ))
 
-    @api.model
-    def create(self, values):
-        if values.get('template_id') and not (values.get('body') or values.get('subject')):
-            template = self.env['mail.template'].browse(values['template_id'])
-            if not values.get('subject'):
-                values['subject'] = template.subject
-            if not values.get('body'):
-                values['body'] = template.body_html
-        return super(SlideChannelInvite, self).create(values)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if vals.get('template_id') and not (vals.get('body') or vals.get('subject')):
```

Comment:
```suggestion
            if vals.get('template_id') and not (vals.get('body') and vals.get('subject')):
```
?
It seems the previous logic was wrong :thinking:

Reply:
I asked to @awa-odoo to take a look :)

Reply:
Maybe it's still a bit too early and I have not had my coffee but it seems correct to me?

"If we have a template and we don't have either the body or the subject, then we replace the subject with the template's subject if it's empty and the body with the template's body if it's empty".

What do you think?

Reply:
It is an "or", as it is used to know if we have to browse the template in case "any" value is missing.